[DOC-12759]: Feedback on Auto-Compaction | Couchbase Docs

RayOffiah opened the pull request:
Updated text and changed a screenshot to note the addition of Magma buckets to auto-compaction.

RayOffiah committed:
[DOC-12759]: Feedback on Auto-Compaction | Couchbase Docs

Updated text and screenshot to note that auto-compaction also covers the Magma buckets.

RayOffiah committed:
[DOC-12759]: Feedback on Auto-Compaction | Couchbase Docs

Updated text and screenshot to note that auto-compaction also covers the Magma buckets.

Added REST setting for magmaFragmentationPercentage

mattyhall commented on modules/manage/pages/manage-settings/configure-compact-settings.adoc:
```diff
@@ -250,38 +250,32 @@ Formatted, this might appear as follows:
     "databaseFragmentationThreshold": {
       "percentage": 30,
-      "size": 536870912
+      "size": "undefined"
```
Were these taken from a run of couchbase server? I've not seen "undefined" returned before

RayOffiah replied:
Mmm. 🤔 

Yes they were taken from my local installation. I'll have another look. 

It 7.6.2 Community edition.

RayOffiah replied:
Fixed.

RayOffiah committed:
[DOC-12759]: Feedback on Auto-Compaction | Couchbase Docs

Updated text and screenshot to note that auto-compaction also covers the Magma buckets.

Added REST setting for magmaFragmentationPercentage
